.github/workflows/Publish-Package.yml

```
@@ -0,0 +1,57 @@
+name: Publish Package
+
+on:
+  push:
+    branches: [master]
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+*" # Push events to matching v*, i.e. v1.0, v20.15.10
```

comment and regex don't match:
https://regex101.com/r/qZ5y8W/1

```suggestion
      - "v[0-9]+.[0-9]+*" # Push events to matching v*, i.e. v1.0, v20.15.10
```

---

I'm tempted to just remove the comment since it's an artifact from previous testing I forgot to remove. 

As far as the wildcard, it's meant to match the poetry version with a 'v' prepended. Could/would our poetry version ever just be major+minor without the patch? 

I tried to be able to pick up any semver we would reasonably want without having to resort to `- "v*"`

---

I'ld also be good with this:
(I doubt poetry will ever release without a patch number, as that wouldn't comply with standard python versioning)

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+*" # Push events to matching v*, i.e. v1.0, v20.15.10
      + "v[0-9]+.[0-9]+.[0-9]+*" # Push events to matching v*, i.e. v1.0.0, v20.15.10
```